applications/ContactStructuralMechanicsApplication/tests/ALM_frictionless_contact_test_2D/mesh_moving_matching_test_parameters.json

```diff
@@ -182,6 +182,7 @@
             "check_variables"     : ["DISPLACEMENT_X", "DISPLACEMENT_Y"],
             "input_file_name"     : "ALM_frictionless_contact_test_2D/mesh_moving_matching_test_results.json",
             "model_part_name"      : "CONTACT_Contact_Auto1",
+            "tolerance"            : 3e-3,
```

Comment:
why don t you leave defaults?

Reply:
Because with the change it converges slightly different for the case of only one LM and LM by components (faster in this case), giving a slightly different result